### ISSUE TITLE
Crashes when influx not correct

### ISSUE THREAD

**Issue by vingerha** [johanmeijer/grott#393]
When influx is setup but is no longer valid (e.g. I deleted the influx db) then the whole setup fails, nothing is pushed to MQTT nor forwarded to growat server. Suggest to 'catch' this error

**Comment by johanmeijer** [comment]
Yes that has been a discussion before. If grott completely stops in case  of a failure the data is not sent to growatt server and stored in the datalogger to be sent later. 

Maybe I can create a parameter that the you can choose to ignore errors and continue processing.

**Comment by vingerha** [comment]
Something like that would indeed be nice, a 'simple' try/except?

**Comment by johanmeijer** [comment]
Question does Grott chrashes or only stops sending records?

**Comment by vingerha** [comment]
Indeed, I forgot to specify. The container kept running but showed erross in the log related to the influxdb. Nothing went to MQTT or Growattserver
